### PR TITLE
Fix FieldMap to include constants for the new line sensor system

### PR DESCRIPTION
**Pull request by pak-plau**

### CODE REVIEW

File: src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableBlueQuadrant.java [hunk 1]
```diff
@@ -6,20 +6,11 @@
...
-    public double getDistanceFromAutoLineToNullTerritory() {
-        return SIDE_CLOSEST_TO_SCORING_TABLE_BLUE_QUADRANT_DISTANCE_FROM_AUTO_LINE_TO_NULL_TERRITORY;
-    }
 
+    //Scale auton where the bot and the scoring scale are on the same side
     @Override
-    public double getDistanceToMoveBackwardBeforeTurn() {
-        return SIDE_CLOSEST_TO_SCORING_TABLE_BLUE_QUADRANT_DISTANCE_TRAVEL_BACKWARDS;
+    public double getTotalDistanceToTravelToReachNullLine() {
+        return SIDE_CLOSEST_TO_SCORING_TABLE_BLUE_QUADRANT_TOTAL_DISTANCE_TO_TRAVEL_TO_REACH_NULL_TERRITORY_LINE;
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableBlueQuadrant.java]: Why don't we just call this `BLUE_SCORING_TABLE_SIDE....`?

File: src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableBlueQuadrant.java [hunk 2]
```diff
@@ -53,30 +40,34 @@ public double getDegreeOfAngleToTurnToReachPlatformZone() {
...
     @Override
-    public double getDistanceToDriveOutAfterPlatformZoneEdge() {
-        return SIDE_CLOSEST_TO_SCORING_TABLE_BLUE_QUADRANT_DISTANCE_TO_DRIVE_OUT_AFTER_PLATFORM_ZONE;
+    public double getDistanceFromStartingPointToOtherPlatformZoneEdge() {
+        return SIDE_CLOSEST_TO_SCORING_TABLE_BLUE_QUADRANT_DISTANCE_FROM_STARTING_POINT_TO_OTHER_PLATFORM_ZONE_EDGE;
     }
 
     @Override
-    public double getDegreeOfAngleToTurnToReachScaleSide() {
+    public double getDegreeOfAngletoTurnToReachScaleSide() {
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableBlueQuadrant.java]: Why don't we just call it `getAngleToTurnToReachScaleSide()`?

File: src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableRedQuadrant.java [hunk 3]
```diff
@@ -6,77 +6,67 @@
  * change, we can adapt quickly. All measurements are in inches for encoder purposes.
  */
 public final class FieldMapSideClosestToScoringTableRedQuadrant implements FieldMapInterface{
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/FieldMapSideClosestToScoringTableRedQuadrant.java]: RedScoringTableQuadrant

**Review by PrathamRawat — COMMENTED** [review]
I think the variables need simpler names.

File: src/org/usfirst/frc/team694/robot/FieldMapMiddleFarFromScoringTableQuadrant.java [hunk 4]
```diff
@@ -5,14 +5,14 @@
...
     //These distances is us starting on the left side of the field
     //all measurements are in inches or degrees if specified 
     //anything with // could be changed due to the actual field measure  
     //These are the measurements that need to be measured out before match.
 
-    public static final int SIDE_CLOSEST_TO_SCORING_TABLE_QUADRANT_DISTANCE_FROM_NULL_TERRITORY_TO_NULL_BUMP = 36; //
-    public static final double SIDE_CLOSEST_TO_SCORING_TABLE_QUADRANT_DISTANCE_FROM_BORDER_TO_SCALE_EDGE = 71.57; //
-}
\ No newline at end of file
+    public static final int MIDDLE_FAR_FROM_SCORING_TABLE_QUADRANT_DISTANCE_FROM_NULL_TERRITORY_TO_NULL_BUMP = 36; //
```
Comment (PrathamRawat) [inline src/org/usfirst/frc/team694/robot/FieldMapMiddleFarFromScoringTableQuadrant.java]: This seems a little confusing, but as long as it makes sense to the people measuring the field.

**Review by TacoTechnica — APPROVED** [review]
welp

**Pull request merged**